### PR TITLE
subnet_range variable

### PR DESCRIPTION
**Pull request by geekbass**
adding the ability for the user to be able to assign a custom subnet range from this module. This is defaulted at the infrastructure level if not defined. This was brought to light while doing testing for multi-region.

### CODE REVIEW

**Comment by geekbass** [comment]
looks like the infrastructure module was bumped to 0.1.10. This build will continue to fail until this version is released for that module.

**Comment by jbomaileva** [comment]
worked fine for us. Thanks !

File: variables.tf [hunk 1]
```diff
@@ -215,3 +215,8 @@ variable "cluster_name_random_string" {
   description = "Add a random string to the cluster name"
   default     = false
 }
+
+variable "subnet_range" {
+  description = "Private IP space to be used in CIDR format"
+  default     = ""
```
Comment (fatz) [inline variables.tf]: i think this needs to clone the default. Rebasing to v0.2 @geekbass can you change the default here

Comment (bernadinm) [reply variables.tf]: This is now resolved with the recent changes that Weston made

**Pull request merged**